src/components/Radio.tsx

```diff
@@ -26,7 +26,7 @@ export const StyledInput = styled.input<{isDisabled?: boolean}>`
   color: ${colors.palette.pale};
   width: 2rem;
   height: 2rem;
-  border: 1px solid currentColor;
+  border: 1px solid #767676;
```

Comment:
Is this value from UX? Like is this going to match new Figma stuff?

Reply:
victoria requested it [here](https://openstax.atlassian.net/browse/DISCO-414?focusedCommentId=12202). I think she's maybe referring to the browser default since I don't see us using that color anywhere.

@vcstax do you think we'll use this color elsewhere? if so I can add it to the theme palette. if not maybe we want to use a gray that's already in there?

Reply:
@vcstax [here is the palette.](https://github.com/openstax/ui-components/blob/main/src/theme/palette.ts) we can add `#767676` or just use `neutralThin`. If we want we could also apply the same to checkboxes so we don't rely on browser defaults.